### DISCUSSION TITLE
Is there a stable Docker tag which is bound to a version range, but still gets updates?

### DISCUSSION THREAD

**Discussion by kaykayehnn** [firefly-iii/firefly-iii#7119] [Firefly III] [answered]
I'm looking for a way to pin my installation to only update to patch or minor releases automatically. Currently the docs suggest using the `latest` tag which is good if you always want to have the latest version, but since major versions can contain [breaking changes](https://github.com/firefly-iii/firefly-iii/releases/tag/v6.0.0) I'd prefer to do these updates manually.

After having looked through the tags history in Dockerhub, it seems like each release gets its own tag and there aren't tags which get updated continuously. Also the `version-6.0` tag has the same hash as `version-6.0.0-beta.1` which would be an issue, since I'm only looking for stable releases.

If it would be possible to have a tag such as `version-6` or `version-6.1` which gets updated when a new stable version is released that would be great

**Comment by JC5** [comment, marked as answer]
I noticed the same and will be fixing that soon. The version tags should not include betas or alphas.

**Reply by JC5** [reply]
This is now in place. Both `version-6` and `version-6.0` will contain stable releases only.

**Reply by kaykayehnn** [reply]
Awesome! Thank you for the lightning fast solution 🙌
